libkbfs/disk_limits_unix.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+// +build !windows
+
+package libkbfs
+
+import (
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+)
+
+// getDiskLimits gets the disk limits for the logical disk containing
+// the given path.
+//
+// TODO: Also return available files.
+func getDiskLimits(path string) (availableBytes uint64, err error) {
```

Comment:
Just the fact that it has platform-specific bits, and the fact that it's morally its own package (and may be in the future).